release-notes/release-notes52.md

```diff
@@ -62,7 +62,13 @@ The new `toolbar_groups` setting and `addGroupToolbarButton` registry API can be
 For information on using the `toolbar_groups` setting, see: [User interface options - `toolbar_groups`]({{ site.baseurl }}/configure/editor-appearance/#toolbar_groups).
 For information on using the `addGroupToolbarButton` API, see: [Types of toolbar buttons - Group button]({{ site.baseurl }}/ui-components/typesoftoolbarbuttons/#grouptoolbarbutton).
 
-### Updated the table icons
+### Updated the default TinyMCE table icons
+
+The {{site.productname}} icons for table-related toolbar buttons have been updated to better match the icons used elsewhere in the product.
+
+This change applies to the default {{site.productname}} theme, `silver`.
```

Comment:
I assume this is just so as to avoid confusion for anyone using the old `mobile` theme? If not this seems unnecessary?

Reply:
the icons wouldn't change on the icon packs.

Reply:
We shouldn't mention `theme` then and instead should just say the default icon pack.

Reply:
changed